### ISSUE TITLE
Security group templates start wide open

### ISSUE THREAD

**Issue by bdirito** [terraform-aws-modules/terraform-aws-security-group#2]
```
variable "ingress_cidr_blocks" {
  description = "List of IPv4 CIDR ranges to use on all ingress rules"
  default     = ["0.0.0.0/0"]
}

variable "ingress_ipv6_cidr_blocks" {
  description = "List of IPv6 CIDR ranges to use on all ingress rules"
  default     = ["::/0"]
}
```
example from postgresql module. It seems acceptable to have egress be open by default but it should NOT be open by default on ingress. Didnt check all of the other predefines; but they also appear to have the same issue.

**Comment by antonbabenko** [comment]
I was actually thinking about this myself when I was working on this module, but forgot to review it before pushing. THANKS for looking into the code!

I think we need to require `ingress_cidr_blocks` and `ingress_cidr_blocks` and don't assume anything there. Agree?

**Comment by antonbabenko** [comment]
Fixed in release [v1.1.0](https://github.com/terraform-aws-modules/terraform-aws-security-group/releases/tag/v1.1.0)
